### PR TITLE
Enable registering logo image

### PR DESCRIPTION
**Pull request by eunos-1128**
#141

I added Rust logo for Jupyter launcher page.

image

### CODE REVIEW

File: crates/irust_repl/irust_kernel/irust_kernel/resources/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import pathlib
+
+
+LOGO_PATH = pathlib.Path(__file__).resolve().parent / "logo-svg.svg"
```
Comment (sigmaSd) [inline crates/irust_repl/irust_kernel/irust_kernel/resources/__init__.py:4]: Does this have to be in its own file? if it can be in install.py its better

**Review by sigmaSd — COMMENTED** [review]
some comments

**Comment by sigmaSd** [comment]
Thanks for the pr, that's a good addition

File: crates/irust_repl/irust_kernel/irust_kernel/resources/__init__.py (hunk 1 above)
Comment (eunos-1128) [reply crates/irust_repl/irust_kernel/irust_kernel/resources/__init__.py:4]: I think it can be also in install.py and either will work.

**Pull request merged**